frontends/concrete-rust/concrete-keygen/webapp/README.md

```diff
@@ -0,0 +1,42 @@
+# Chunked KeyGen WebApp
+
```

Comment:
Maybe just warn that is just a tool example to demonstrate the chunked key generation